lib/web_ui/test/keyboard_converter_test.dart

```diff
@@ -530,30 +537,41 @@ void testMain() {
       keyDataList.add(key);
       return true;
     });
+    MockKeyboardEvent event;
 
     // Same layout
-    converter.handleEvent(keyDownEvent('KeyA', 'a'));
-    converter.handleEvent(keyDownEvent('KeyA', 'a'));
-    converter.handleEvent(keyUpEvent('KeyA', 'a'));
-    converter.handleEvent(keyUpEvent('KeyA', 'a'));
+    event = keyDownEvent('KeyA', 'a');
```

Comment:
What's the reason for splitting these into two lines each everywhere instead of one?  To me, they seem a lot more readable as one line.

Reply:
It was used to hold the created event object and verify the `defaultPrevented`. I have changed it by adding a `lastDefaultPrevented` static field to the mock class.